tree/master/packages/extension/public/locales/pl/translation.json

```
@@ -0,0 +1,162 @@
+{
+	  "Copied": "Skopiowane",
+	  "<unknown>": "<nieznane>",
+	  "click to select or drag and drop the file here": "kliknij, aby wybrać lub przeciągnij i upuść plik tutaj",
+	  "{{name}} ({{size}} bytes)": "{{name}} ({{size}} bajtów)",
+	  "... loading ...": "... Ładowanie ...",
+	  "Generated 12-word mnemonic seed:": "Wygenerowane 12-wyrazowe ziarno mnemoniczne:",
+	  "Copy to clipboard": "Skopiuj do schowka",
+	  "Back": "Z powrotem",
+	  "Create new account (root or derived)": "Utwórz nowe konto (główne lub pochodne)",
+	  "Create new account": "Stwórz nowe konto",
+	  "Import account from pre-existing seed": "Importu konto z wcześniej istniejącego materiału siewnego",
+	  "Restore account from backup JSON file": "Przywróć konto z zapasowego pliku JSON",
+	  "Attach external QR-signer account": "Dołącz zewnętrzne konto osoby podpisującej kod QR",
```

```suggestion
	  "Attach external QR-signer account": "Dołącz zewnętrzne konto QR",
```
Wiem, że to pomija tłumaczenie `signer`, ale np. w kolejnych liniach też używamy frazy "konta QR", więc zachowałbym spójność.